fractal_tasks_core/tasks/create_ome_zarr_multiplex.py

```diff
@@ -19,8 +19,8 @@
 
 import pandas as pd
 import zarr
-from anndata._io.specs import write_elem
 from pydantic.decorator import validate_arguments
+from zarr.errors import ContainsGroupError
```

Comment:
Is importing `ContainsGroupError` still necessary in create ome zarr multiplex? Looks like a leftover, as it's not called afterwards (only in the imported helper functions)

Reply:
Nevermind, saw where it was needed :)